ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1057,10 +1057,13 @@ static PrjFS_Result HandleFileNotification(
         request->pid,
         request->procname,
         isDirectory,
+        placeholderFile,
         notificationType,
         nullptr /* destinationRelativePath */);
     
-    if (result == 0 && placeholderFile && PrjFS_NotificationType_PreConvertToFull == notificationType)
+    if ((result == PrjFS_Result_Success) &&
```

Comment:
@wilbaker, I'm not sure how the result==0 would have worked.  Thoughts?

Reply:
@jeschu1 agreed, this looks like it was a bug.  Thanks for catching this!